CMakeLists.txt

```diff
@@ -92,12 +114,48 @@ endif()
 target_compile_definitions(ccls PRIVATE
                            DEFAULT_RESOURCE_DIRECTORY=R"\(${Clang_RESOURCE_DIR}\)")
 
+
+### use local installed RapidJSON or FETCH from git via cmake
+find_package(RapidJSON QUIET)
```

Comment:
This locates the system rapidjson (say /usr/include/rapidjson) if exists?

Reply:
Yes: https://github.com/Tencent/rapidjson/blob/master/RapidJSONConfig.cmake.in#L13